ports/stm32/mboot/README.md

```diff
@@ -209,6 +219,33 @@ and signed firmware, and can be deployed via USB DFU, or by copying it to the de
 internal filesystem (if `MBOOT_FSLOAD` is enabled). `firmware.dfu` is still unencrypted
 and can be directly flashed with jtag etc.
 
+Retrieving the mboot version in micropython
+--------------------------------------
+The following python code retrieves the version of mboot currently on the device:
+```python
+import stm
+
+def get_mboot_version(include_opts=True):
```

Comment:
I suggest putting this code in `fwupdate.py`, and adding arguments for the base, length and version length parameters (with defaults).